packages/TelegramClient-Tests.package/TCTAcceptanceTests.class/instance/testPrivateChatHasNoDescription.st

```diff
@@ -0,0 +1,17 @@
+testing
+testPrivateChatHasNoDescription
+
+	| titleBar groupChat chatInfoPage |
+	self wantsToTest: (TCUMain newWithCore: self core).
+	
+	[self core chats anySatisfy: [:c | c isPrivate]]
+		whileFalse: [6 seconds wait.].
+	groupChat := (self core chats detect: [:chat | chat isPrivate]).
+	
+	(self subject findByCriteria: [ :morph | (morph respondsTo: #chatID) and: [morph chatID = groupChat id]]) click.
+	titleBar := (self subject findByClass: TCUChatWindow) morphs collect: [:chatWindow | chatWindow titleBar].
+	(self subject createWrapperFor: titleBar) click.
+
+	chatInfoPage := (self subject findByClass: TCUChatInfoPage) morphs anyOne.
+	self forceSaveScreenshotOf: chatInfoPage as: 'testPrivateChatHasNoDescription' in: self class defaultTestResultPath.
+	self assert: 134349208 equals: (self computeHashOf: (self takeScreenshotOf: chatInfoPage)).
```

Comment:
maybe put the constant into an accessor

Reply:
I know magic numbers are a thing but these hashes are so unique and a core part of the test that I'm not sure whether we have a benefit from writing an accessor for each hash